### ISSUE TITLE
^few things^

### ISSUE THREAD

**Issue by ahjolinna** [fabianalexisinostroza/Antu#1]
could you change the 'show desktop' widget icon...it's really distracting at least if u have KSysGuard icon/app next to it in the plasma panel.

Also could make few folders, kdenlive, MEGAsync, PlayOnLinux's virtual drivers, [dwhelper](https://addons.mozilla.org/en-US/firefox/addon/video-downloadhelper/) and if possible tv-show, movies...oh and Anime :D use your imagination what the icon should be on them ...oc you could use the general/boring videos icon...but why :P ?!

There is also few apps that needs icons..at least... qmmp, ROCCAT, KDE-Services, nvidia-settings, pdfeditor, qcomibook, xperia-theme-creator, DiskMonitor, yacreader...
I always recommend to look at the [Nitrux project](https://github.com/NitruxSA/nitrux-icons/) they have the most complite set of icons...doesn't matter what you think about their work...it's still good "icon-database" for you

Anyway have you thought about making a sddm, Ksplash or plymouth themes? (or maybe even a bomi-player)

**Comment by fabianalexisinostroza** [comment]
I will try to make these ideas.
Plymouth themes ... i created, check KDE-Look
About sddm and ksplash i don't know how to make, only can make concepts

**Comment by ahjolinna** [comment]
I found some more job for you :D 

these have same icons, I don't like that:
_try some other color('s) or something_

1. 'Calligra' and 'Libreoffice'
2. 'KMail' and 'KMail Header Theme Editor' and 'Cloud Storage Manager'
3. 'Contact Print Theme Editor' and 'Contact Theme Editor' and 'KAddressBook' and 'Kontact'
4. 'UXTerm' and 'XTerm'

these misses icons:
PIM Setting Exporter, WPS, T-lasku, Sieve Editor, Baka-MPlayer, MKV Extractor Qt, QMPlay2, aLinkCreator(amule), MEGASync, Messenger (FB chat app), FontForge, SSH AskPass, X-Utilities, PeaZip, VapourSynth Editor

btw. I just added your theme('s) to ccr: [icon](https://chakraos.org/ccr/packages.php?ID=7520) [plasma](https://chakraos.org/ccr/packages.php?ID=7523)
unfortunately ccr doesn't yet support package renaming, so I couldn't replace the old ones.
I will oc maintain them in ccr until chakra has moved them to the repo
